CONTRIBUTING.md

```diff
@@ -79,6 +79,15 @@ View changes made by the release script
 # see also dagster-cloud-action-test
 ```
 
+It is recommended to test the release with both Serverless and Hybrid deployments.
+
+Serverless:
+- GitLab: Deploy to a test organization in Dagster+ after updating the image version in the CI/CD file.
+- GitHub: Deploy to a test organization in Dagster+ with both PEX and docker deployments.
+
+Hybrid:
+- GitHub: Create a new branch in the [Hooli repo](https://github.com/dagster-io/hooli-data-eng-pipelines) to update the release version. Push your changes and open a PR to test the deployment to Dagster+.
```

Comment:
update the tags in the github action to use then release version right?

Reply:
Right. I've updated the sentence in [88081d6](https://github.com/dagster-io/dagster-cloud-action/pull/186/commits/88081d69127745efa15b2832ab6eb6fbed9d103b)